frontend/src/scenes/feedback/Feedback.tsx

```diff
@@ -0,0 +1,384 @@
+
+export function ExpandableSection({
+    header,
+    children,
+    expanded,
+    setExpanded,
+}: {
+    header: JSX.Element
+    children: JSX.Element
+    expanded: boolean
+    setExpanded: (expanded: boolean) => void
+}): JSX.Element {
+    return (
+        <div className="rounded border p-4 my-4">
+            <div
+                className="flex cursor-pointer"
+                onClick={() => {
+                    setExpanded(!expanded)
+                }}
+            >
+                <LemonButton
+                    className="inline-flex items-center justify-center mr-2 mb-2"
+                    status="stealth"
+                    noPadding={true}
+                    onClick={() => setExpanded(!expanded)}
+                    icon={expanded ? <IconUnfoldLess /> : <IconUnfoldMore />}
+                    title={expanded ? 'Collapse' : 'Expand'}
+                />
+                <div className="inline-flex">{header}</div>
+            </div>
+            <div>{expanded && children}</div>
+        </div>
+    )
+}
+
+export function FeedbackInstructions(): JSX.Element {
+    // const formKey = 'newFeedbackEvent'
+    // const eventName = 'Feedback Sent'
+    // const feedbackProperties = ['$feedback']
+
+    const { expandedSection } = useValues(feedbackLogic)
+    const { setExpandedSection } = useActions(feedbackLogic)
+
+    return (
+        <div className="max-w-200">
+            <ExpandableSection
+                header={<h2 className="text-xl">Set up the feedback widget</h2>}
+                expanded={expandedSection(0)}
+                setExpanded={(expanded) => setExpandedSection(0, expanded)}
+            >
+                <div>
+                    <p className="text-sm italic">
+                        The feedback widget is the quickest way to collect feedback from your users.
+                    </p>
+                    <div>
+                        <div>
+                            <div className="text-lg">1. Turn on the feedback widget</div>
+                            <div className="ml-4 my-4">
+                                <LemonButton
+                                    onClick={() => {
+                                        window.open(urls.projectAppSearch('Feedback Widget'), '_blank')
+                                    }}
+                                    type="primary"
+                                >
+                                    Feedback widget
+                                </LemonButton>
+                            </div>
+                        </div>
+                        <div>
+                            <div className="text-lg">2. Enable site apps</div>
+                            <div className="ml-4 my-4">
+                                <CodeSnippet language={Language.JavaScript} wrap>
+                                    {`posthog.init('YOUR_PROJECT_API_KEY', {
```

Comment:
nit: formatting looks weird here - maybe we can make this a const somewhere?

Reply:
great idea about doing const